Components/Avatar/avatar.css

```diff
@@ -0,0 +1,102 @@
+@import url('../component.css');
+@import url('../theme.css');
+@import url('../../documentation/style.css');
+
+
+/* avatar  */
+.avatar{
```

Comment:
keep a space between the selector and opening bracket.

Reply:
Thankyou for your review. I have made the changes to the CSS files.